static/js/index.js

```diff
@@ -363,12 +363,16 @@ function renderStudent(studentObj) {
   var popupContainer = document.getElementById("popupContainer");
   var email = studentObj.email;
   email = email.toLowerCase();
+  var preferredname = studentObj.preferredname;
```

Comment:
does `preferredname` ever get set? it looks like the Python code just changes `firstname` to be the preferred name.

Reply:
Yes, I think you are right. We don't need this pr.